CHANGES.md

```diff
@@ -1,6 +1,289 @@
+
+released December 18, 2025
+
+Highlights (see the sections that follow for details)
+-----------------------------------------------------
+* Chapel's first release since becoming an HPSF / Linux Foundation project  
+  (see https://hpsf.io/blog/2025/hpsf-welcomes-chapel/)
+* a new `--vector-library` flag to simplify using optimized back-end libraries
+* stack traces for errors by default in most configurations and builds
+* improved debugging, esp. when using multiple locales and the global namespace
+* many Mason advances, including build improvements for multi-language packages
+* several Linux package improvements in terms of flexibility and features
+* many other improvements in terms of features, bug fixes, docs, error msgs
+
+Compiler Flags
+--------------
+* added a new `--vector-library` flag to enable using vectorization libraries  
+  (see https://chapel-lang.org/docs/2.7/usingchapel/man.html#man-vector-library)
+* added `--debug-safe-optimizations-only` to disable key opts. for debugging  
+  (see https://chapel-lang.org/docs/2.7/usingchapel/man.html#man-debug-safe-optimizations-only)
+* changed the long form of `-g` to `--debug-symbols`  
+  (see https://chapel-lang.org/docs/2.7/usingchapel/man.html#man-debug-symbols)
+* changed `--debug` to imply `-g` and `--debug-safe-optimizations-only`  
+  (see https://chapel-lang.org/docs/2.7/usingchapel/man.html#man-debug)
+* added `--print-passes-memory` to report memory used by compiler passes  
+  (see https://chapel-lang.org/docs/2.7/usingchapel/man.html#man-print-passes-memory)
+* removed all previously deprecated compiler flag environment variables  
+  (e.g., `CHPL_NO_CHECKS`, `CHPL_NO_NIL_CHECKS`, `CHPL_NO_CODEGEN`, etc.)
+
+Debugging Improvements
+----------------------
+* added wide pointer resolution to `chpl-parallel-dbg`
+* improved the debugger's pretty printing of `uint(8)` and `int(8)`
+* hid internal module names for types like `string` and tuples
+* added debug information for complex numbers, syncs, and atomics
+* improved generated debug info, especially for classes and wide pointers
+* updated the debug info for `ref`s to be treated as references, not pointers
+* removed compiler-inserted temporary variables from debug info where possible
+
+Tool Improvements
+-----------------
+* improved the ergonomics of building `mason`  
+  (see https://chapel-lang.org/docs/2.7/tools/mason/start/installation.html)
+* added a "prerequisites" capability to Mason to support interoperability  
+  (see https://chapel-lang.org/docs/2.7/tools/mason/guide/prereqs.html)
+* improved the integration of `mason` into `chpl-language-server`
+* improved the performance of `mason` by building it with optimizations
+* fixed a bug where `mason` failed to detect changes in submodules
+* made bug fixes and other minor improvements to `mason test`
+* started to modernize Mason's integration with Spack
+* started to improve Mason's logging capabilities
+* fixed incorrect call inlays for method calls
```

Comment:
Since all the bullets prior have been about mason, and since we don't say "in CLS" or something here, this bullet is, to me, very confusing. I think saying "in `chpl-language-server`" would clarify it, and looks like it might fit.